ext/hal/ti/simplelink/source/ti/devices/cc13x2_cc26x2/driverlib/uart.c

```diff
@@ -36,7 +36,7 @@
 *
 ******************************************************************************/
 
-#include "uart.h"
+#include <periph/uart.h>
```

Comment:
This should not be changed. The file is referring to ./ext/hal/ti/simplelink/source/ti/devices/cc13x2_cc26x2/driverlib/uart.h, not the one in ./include